src/Language/PureScript/Docs/AsMarkdown.hs

```diff
@@ -62,6 +64,21 @@ codeToString = outputWith elemAsMarkdown
   elemAsMarkdown (Keyword x) = x
   elemAsMarkdown Space       = " "
 
+fixityAsMarkdown :: P.Fixity -> Docs
+fixityAsMarkdown (P.Fixity associativity precedence) = do
+  tell' $ concat [ "associativity: **"
```

Comment:
I think using English words makes it a bit easier to understand. Given that this concept exists outside of PureScript and Haskell, I would guess that more people understand the term "left-associative" than "infixl", for example. And if you do understand "infixl", you probably also understand "left-associative".

If you were referring to putting it inline with the code like Haddock does, I can't see a good way of doing that in Markdown that preserves some kind of visual separation between the type signature and the fixity declaration, unfortunately.